embulk-core/src/main/java/org/embulk/exec/BulkLoader.java

```diff
@@ -330,7 +330,7 @@ public ResumeState buildResumeState(ExecSession exec) {
             List<Optional<TaskReport>> inputTaskReports = (inputTaskStates == null) ? null : getInputTaskReports();
             List<Optional<TaskReport>> outputTaskReports = (outputTaskStates == null) ? null : getOutputTaskReports();
             return new ResumeState(
-                    exec.getSessionExecConfig(),
+                    exec.newConfigSource().set("transaction_time", exec.getTransactionTimeString()),
```

Comment:
looks like `exec.getSessionExecConfig()` already doing the same thing

Reply:
Yeah, but `Exec.getSessionExecConfig` is now `@Deprecated`, and we intend it to be removed finally. That's there only for compatibilify for a while. Rather than `@SuppressWarnings("deprecation")`, I chose to do the same here.

Reply:
oops, true:) I missed the annotation